wallet/how-to/display/tokens.md

```diff
@@ -19,11 +19,14 @@ You can improve the security and experience of displaying your
 `wallet_watchAsset` provides a friendly interface that prompts users to register tokens to their
 MetaMask wallet, without having to interact with contract addresses.
 
-:::tip Displaying NFTs
-With `wallet_watchAsset`, you can prompt users to add their NFTs even when they have NFT
-autodetection disabled.
-Moreover, NFT autodetection only detects NFTs on Ethereum Mainnet.
-With `wallet_watchAsset`, users can add NFTs from other networks.
+:::tip
+With `wallet_watchAsset`, you can prompt users to add their ERC-20 tokens and NFTs even when they
+have token autodetection or NFT autodetection disabled.
+
+Moreover, token autodetection only detects ERC-20 tokens on
```

Comment:
Moreover might be extraneous. Maybe remove for brevity?
```suggestion
Token autodetection only detects ERC-20 tokens on
```

Reply:
It's meant to emphasize an additional benefit. Would "also" be better?